eventlet/support/afinet.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test availabilty of AF_INET6 on a machine
+
+Uses socket API to determine configuration
+"""
+
+from socket import *
+
+def check_ipv6_lo_addr():
+    try:
+        sock = socket(AF_INET6, SOCK_STREAM)
+        sock.bind(('::1', 0))
+        return True
+    except:
```

Comment:
`except:` is absolutely prohibited. We do not hide unexpected errors. Consider there was `socket.bindwrwongname(...)`, this code would hide error. On Python2 this is double error, because `except:` also catches `GeneratorExit`, `SystemExit`, `KeyboardInterrupt `.

https://docs.python.org/2/library/exceptions.html